src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -153,7 +153,7 @@ public IEnumerable<NodeSearchElementViewModel> FilteredResults
             }
             set
             {
-                filteredResults = value;
+                filteredResults = ToggleSelect(value);
```

Comment:
Yes, I added `FirstItemIsSelectedAfterSearch` and `NoItemsIsSelectedAfterSearch` tests.